microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+
+
+<!-- ------------------------------ -->
+#### Sensitive information that's not sent
+
+<!-- The Privacy settings have been updated from what is described here (assume comment means h2 section) -->
+
+If Microsoft Edge detects typing in the Address bar that might contain sensitive information, Microsoft Edge doesn't send the typed text.  Sensitive information includes:
+* Authentication credentials.
+* Local file names.
+* URL data that's normally encrypted.
+
+
+<!-- ------------------------------ -->
+#### Enabling collecting Address bar diagnostic data
+
+You can configure Microsoft Edge to collect diagnostic data about the Address bar.  Collected data includes the number of offered queries for all search providers.  To enable collecting diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, and then in **Personalize your web experience**, turn on the setting **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services**.
+
 
-You can configure Microsoft Edge to collect diagnostic data about the address bar. Collected data includes the number of offered queries for all search providers. Go to `edge://settings/privacy`. In **Personalize your web experience** turn on the **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services** setting.
+<!-- ------------------------------ -->
+#### Deleting data
+
+Typed characters and the websites that you visit are stored locally on the device, per profile.  To delete that data, go to `edge://settings/clearBrowserData`, and then in the **Clear browsing data** window, select the **Browsing history** checkbox.  Then click the **Clear now** button.
 
-Typed characters and the websites you visit are stored locally on the device per profile.  To delete the data, go to `edge://settings/clearBrowserData`. In the **Clear browsing data** window, select the **Browsing history** checkbox, and select the **Clear now** button.
+If Bing is your default search provider and you are signed in, you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the Address bar: go to `edge://history`, and then select **Clear browsing data**.  You can delete the data that Microsoft collects from the Address bar and search suggestions features on Windows 10 and later: open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then in **Delete diagnostic data**, select **Delete**.<!-- deleting diagnostic data could use rewrite -->  All other data is deleted after 36 months.
```

Comment:
Is this for the right feature? If I was reading this as a user, I'd be confused that I'm deleting Edge data from inside the OS rather than inside Edge.
``
You can delete the data that Microsoft collects from the Address bar and search suggestions features on Windows 10 and later: open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then in **Delete diagnostic data**, select **Delete**.
``

Reply:
@dexterqu2024 - who can tech review to handle this question?  
see rendered article section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3224#deleting-data

Reply:
i added the question as a todo hidden comment in the .md file so it can be handled in a later PR.
representing Markdown w/ comment is awkward here so trying a couple formats @apryzgoda 

result formatted via simple copypaste:
If Bing is your default search provider and you are signed in, you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the Address bar: go to `edge://history`, and then select **Clear browsing data**.  You can delete the data that Microsoft collects from the Address bar and search suggestions features on Windows 10 and later: open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then in **Delete diagnostic data**, select **Delete**.<!-- todo: is this sentence about Start menu for the correct feature?  seems to be about deleting Edge data from within the OS rather than from within Edge --><!-- old comment: deleting diagnostic data could use rewrite -->  All other data is deleted after 36 months.

result formatted within backticks:
```
If Bing is your default search provider and you are signed in, you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the Address bar: go to `edge://history`, and then select **Clear browsing data**.  You can delete the data that Microsoft collects from the Address bar and search suggestions features on Windows 10 and later: open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then in **Delete diagnostic data**, select **Delete**.<!-- todo: is this sentence about Start menu for the correct feature?  seems to be about deleting Edge data from within the OS rather than from within Edge --><!-- old comment: deleting diagnostic data could use rewrite -->  All other data is deleted after 36 months.
```

Reply:
result in backticks w/ linebreaks temp added here (or see fresh Files Changed tab, or latest .md file listing)
```
If Bing is your default search provider and you are signed in, 
you can delete your searches by using the 
[Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  
You can clear your browsing history and delete websites from 
appearing as suggestions in the Address bar: go to `edge://history`, 
and then select **Clear browsing data**.  You can delete the data 
that Microsoft collects from the Address bar and search suggestions 
features on Windows 10 and later: open **Start** > **Settings** > 
**Privacy** > **Diagnostics & feedback**, and then in **Delete 
diagnostic data**, select **Delete**.
<!-- todo: is this sentence about Start menu for the correct 
feature?  seems to be about deleting Edge data from within the 
OS rather than from within Edge -->
<!-- old comment: deleting diagnostic data could use rewrite -->
All other data is deleted after 36 months.
```

Reply:
also added bottom two paragraphs & screenshot, per offline discussion:
![image](https://github.com/user-attachments/assets/9be082d9-15fd-454a-8ac9-bf0b29015741)
@apryzgoda 
Resolving comment, b/c added todo comment for future PR, & info from offline